project.godot

```diff
@@ -21,13 +21,18 @@ config/icon="res://assets/common/icons/application_icon.svg"
 ZoneLayouts="*res://assets/persistence/zone_layouts.gd"
 Savegame="*res://assets/persistence/savegame.gd"
 GameManager="*res://assets/common/scripts/game_manager.gd"
+Consts="*res://assets/common/scripts/consts.gd"
```

Comment:
This should be moved to top of autoload list, or refactored to use static functions/fields